base/replutil.jl

```diff
@@ -444,6 +456,10 @@ function show_method_candidates(io::IO, ex::MethodError, kwargs::Vector=Any[])
             buf = IOBuffer()
             tv = Any[]
             sig0 = method.sig
+            # @Hardcoded: don't display the sysimg.jl method
```

Comment:
I'll note how few tagged-comments, if any, are anything but `TODO` and `FIX`, which isn't very helpful.

I also occurs to me that doing something like `#TAG` would be better in Julia code, since you can search for that pretty easily, plus, you can do `#[\w_]+:?` to search for _any_ tag.